src/ExpressiveInstaller/OptionalPackages.php

```diff
@@ -240,6 +240,7 @@ private static function cleanUp(IOInterface $io, $projectRoot)
         unlink($projectRoot . '/CHANGELOG.md');
         unlink($projectRoot . '/CONDUCT.md');
         unlink($projectRoot . '/CONTRIBUTING.md');
+        unlink($projectRoot . '/phpcs.xml');
```

Comment:
Okay, missed that you were already removing it. However, we need a standard in place when done.

I'm testing the following locally:

- Including a `phpcs.xml.dist` file with the original PSR-2 ruleset.
- Adding a devleopment requirement for `"squizlabs/php_codesniffer": "^2.7"`

This combination should present a usable, minimal phpcs toolchain from the outset, which developers can then customize.